gen/definitions/profile_parcels/service_ipv4_acl.yaml

```diff
@@ -0,0 +1,125 @@
+  - model_name: defaultAction
+    example: drop
+  - model_name: sequences
+    attributes: 
+    - model_name: sequenceId
+      id: true
+      example: 1
+    - model_name: sequenceName
+      example: AccessControlList1
+    - model_name: baseAction
+      exclude_test: true
+      exclude_null: true
+      example: drop
+    - model_name: matchEntries
+      tf_name: conditions
+      attributes: 
+      - model_name: dscp
+        tf_name: dscps
+        example: 16
+      - model_name: packetLength
+        example: 1500
+      - model_name: protocol
+        tf_name: protocols
+        example: 1
+      - model_name: icmpMsg
+        tf_name: icmp_messages
+        example: dod-host-prohibited
+        exclude_test: true
+      - model_name: refId
+        tf_name: source_data_prefix_list_id
+        data_path: [sourceDataPrefix, sourceDataPrefixList]
+        example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+        exclude_test: true
+      - model_name: sourceIpPrefix
+        tf_name: source_data_prefix
+        data_path: [sourceDataPrefix]
+        example: 1.2.3.4/32
+        exclude_test: true
+      - model_name: sourcePorts
+        exclude_null: true
+        attributes: 
+        - model_name: sourcePort
+          tf_name: port
+          example: 8000
+      - model_name: refId
+        data_path: [destinationDataPrefix, destinationDataPrefixList]
+        tf_name: destination_data_prefix_list_id
+        exclude_test: true
+        example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      - model_name: destinationIpPrefix
+        tf_name: destination_data_prefix
+        data_path: [destinationDataPrefix]
+        example: 2.3.4.5/32
+        exclude_test: true
+      - model_name: destinationPorts
+        exclude_test: true
+        exclude_null: true
+        attributes: 
+        - model_name: destinationPort
+          tf_name: port
+          example: 9000
+          exclude_test: true
+      - model_name: tcp
+        tf_name: tcp_state
+        example: syn
+    - model_name: actions
+      attributes: 
+      - model_name: setDscp
+        tf_name: accept_dscp
+        data_path: [accept]
+        example: 60
+      - model_name: counterName
+        tf_name: accept_counter_name
+        data_path: [accept]
+        example: COUNTER_1
+      - model_name: log
+        tf_name: accept_log
+        data_path: [accept]
+        example: false
+        exclude_null: true
+      - model_name: setNextHop
+        tf_name: accept_next_hop
+        data_path: [accept]
+        example: 1.2.3.4
+      - model_name: refId
+        tf_name: accept_mirror_list_id
+        data_path: [accept, mirror]
+        example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+        exclude_test: true
+      - model_name: refId
+        tf_name: accept_policer_id
+        data_path: [accept, policer]
+        example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+        exclude_test: true
+      - model_name: counterName
+        tf_name: drop_counter_name
```

Comment:
either make it "counter_name" on condition that we have drop or call it "drop_counter_name" (if first option not possible)